sources/vega/commands/v1/commands.proto

```diff
@@ -92,6 +92,25 @@ message LiquidityProvisionSubmission {
   string reference = 6;
 }
 
+// Cancel a liquidity provision request
+message LiquidityProvisionCancellation {
+  string id = 1; // this is required
+  string market_id = 2; // this can be remove if we get a nice way to get provisions IDs to market
+}
+
+// Amend a liquidity provision request
+message LiquidityProvisionAmendment {
+  string id = 1; // this is required
+  string market_id = 2; // this can be remove if we get a nice way to get provisions IDs to market
+
+  // From here at least one of the following is required to consider the command valid
```

Comment:
Should this comment be reflected in `checkLiquidityProvisionAmendment` will that be in `core`?

Reply:
I'm sorry, I don't know what you mean. If you are referring to line 106, I have just realised that the validation doesn't check for this so I need to add it.